server/portal/apps/projects/tasks.py

```diff
@@ -0,0 +1,79 @@
+"""
```

Comment:
are these tasks used anywhere?  i couldn't see anything in cepv1.  

Should we just drop this file?